templates/js-cypress/main.js

```diff
@@ -0,0 +1,62 @@
+import { Actor } from 'apify';
+import fs from 'fs';
+import cypress from 'cypress';
+import { globby } from 'globby';
+import log from '@apify/log';
+import 'console.table';
```

Comment:
Just a question, do you really need to import this here?

Reply:
it seems yes, if not there, the summary table is not drawn

<img width="810" alt="Screenshot 2023-03-16 at 13 16 42" src="https://user-images.githubusercontent.com/56041262/225614808-fa24f431-a398-4dff-a794-e612c740c1f1.png">

Reply:
Interesting 🤔